authentication/authentication-core/src/main/java/io/mosip/authentication/core/constant/IdAuthConfigKeyConstants.java

```diff
@@ -158,5 +159,7 @@ private IdAuthConfigKeyConstants() {
 	public static final String USER_PREFFRRED_LANG_ATTRIBUTE_NAME="mosip.default.user-preferred-language-attribute";
 
 	public static final String BIO_SEGMENT_TIME_DIFF_ALLOWED = "authrequest.biometrics.allowed-segment-time-difference-in-seconds";
+	public static final String PREFERRED_LANG_ATTRIB = "mosip.default.user-preferred-language-attribute";
```

Comment:
@LoganathanSekar7627 in identity json, there is a attribute called preferredLang, I think sasi pointed that attribute

Reply:
Fixed.